### PR TITLE
Feature/je 160 remaining

### PR DESCRIPTION
**Pull request by tuj**
* Fixed invoice edit form.
* Added payed by account field to invoice.
* Preview invoice csv from invoice.
* Invoice exported date in list.

### CODE REVIEW

File: bundles/Billing/Entity/Invoice.php [hunk 1]
```diff
@@ -99,6 +99,11 @@ class Invoice
      */
     private $lockedSalesChannel;
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $payedByAccount;
```
Comment (rimi-itk) [inline bundles/Billing/Entity/Invoice.php]: “payed”?! Is this the nautical term (https://en.wiktionary.org/wiki/pay#Verb_2)?

File: bundles/Billing/Resources/assets/pages/HomePage.js [hunk 2]
```diff
@@ -189,7 +189,7 @@ class HomePage extends Component {
                 keyEvent: 'posted',
                 items: invoices
                     .filter((item) => {
-                        return item.recorded === true && item.exportedDate === null;
+                        return item.recorded === true;
```
Comment (rimi-itk) [inline bundles/Billing/Resources/assets/pages/HomePage.js]: This should be `.filter(item => item.recorded === true)` or just `.filter(item => item.recorded)` (if we know that `item.recorded` is a boolean value).

File: bundles/Billing/Resources/assets/pages/HomePage.js [hunk 3]
```diff
@@ -251,6 +251,9 @@ class HomePage extends Component {
                                         <th>{t('home_page.table.project')}</th>
                                         <th>{t('home_page.table.date')}</th>
                                         <th>{t('home_page.table.amount')}</th>
+                                        {tab.keyEvent === 'posted' &&
+                                            <th>{t('home_page.table.exported_date')}</th>
```
Comment (rimi-itk) [inline bundles/Billing/Resources/assets/pages/HomePage.js:249]: Some will be confused by this clever use of `&&`. Is this a recommended use in React?

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Some minor changes required.

File: bundles/Billing/Resources/assets/pages/HomePage.js (hunk 3 above)
Comment (tuj) [reply bundles/Billing/Resources/assets/pages/HomePage.js:249]: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

**Review by rimi-itk — APPROVED** [review]
LGTM.

**Pull request merged**